pkg/skaffold/jib/jib.go

```diff
@@ -30,14 +35,34 @@ func getDependencies(cmd *exec.Cmd) ([]string, error) {
 	}
 
 	// Parses stdout for the dependencies, one per line
-	// TODO(coollog) directories should be expanded recursively
 	lines := strings.Split(string(stdout), "\n")
 	var deps []string
-	for _, l := range lines {
-		if l == "" {
+	for _, dep := range lines {
+		if dep == "" {
+			continue
+		}
+
+		// Resolves directories recursively.
+		info, err := os.Stat(dep)
+		if err != nil {
+			if os.IsNotExist(err) {
```

Comment:
This is modeled after the logic in `changes.go#stat`. I think ignoring non-existent files should be fine here since the files would be ignored by `stat` anyways. The alternative is to just add the file to the `deps` list upon this `os.Stat` error. Thoughts?